### PR TITLE
[Changelog] Add Sprint 97 and Hammer Beta1

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot add_label hammer/no

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,31 +5,33 @@ All notable changes to this project will be documented in this file.
...
+- Move from apache module mod_auth_kerb to mod_auth_gssapi [(#282)](https://github.com/ManageIQ/manageiq-appliance-build/pull/282)
 
 ### Fixed
 - Update vSphere ova [(#278)](https://github.com/ManageIQ/manageiq-appliance-build/pull/278)
+- Increase filesystem sizes [(#232)](https://github.com/ManageIQ/manageiq-appliance-build/pull/232)
+- Set commit_sha just once [(#290)](https://github.com/ManageIQ/manageiq-appliance-build/pull/290)
 
-## Unreleased as of Sprint 91 ending 2018-07-30
+### Removed
+- Remove non-existing "server-policy" yum group [(#268)](https://github.com/ManageIQ/manageiq-appliance-build/pull/268)
```
Comment (carbonin) [inline CHANGELOG.md]: This line should be removed as this fix first appeared in Gaprindashvili-4.

Comment (carbonin) [reply CHANGELOG.md]: How did it even get here? This wasn't in an "unreleased" section before this PR ...

**Comment by simaishi** [comment]
It looks some PRs are missing... e.g. https://github.com/ManageIQ/manageiq-appliance-build/pull/277

**Comment by carbonin** [comment]
@simaishi that should probably be in a Gaprindashvili release though right? I would prefer "fixes" to the changelog be made in separate PRs.

**Comment by simaishi** [comment]
No, that's in Hammer Beta1, so it got missed in the PR that went to Hammer branch too.

**Comment by JPrause** [comment]
What probably happened with PR277, was the Milestone label was not added. As such, it may not have shown up in the list of PRs for that particular sprint.

**Comment by simaishi** [comment]
@JPrause I think Sprint 92 and/or Sprint 93 got missed.. Looks like there are a few other PRs missing from the same time period. Regardless, they can be added now as needed!

**Comment by JPrause** [comment]
@simaishi there are certain PRs (based on labels) that not included in the changelog. This is intentional, and has been in place since before I've been doing the changelogs. So I'd say, let's move on and get these all merged.

**Comment by simaishi** [comment]
@JPrause I really hope we'll change the process to use a label for changelog. Otherwise, the reviewers can't tell why certain PR is included in the changelog and not, and questions like this will come up again. It won't matter if we're going to change the process, but if not, can you at least document how label affects changelogs?

Please do add https://github.com/ManageIQ/manageiq-appliance-build/pull/277 and https://github.com/ManageIQ/manageiq-appliance-build/pull/276 (I'm not sure why this isn't listed as this is labeled as `bug` and has Sprint 92). Those are more important changes (which affect appliances) than PRs like https://github.com/ManageIQ/manageiq-appliance-build/pull/290!

**Comment by JPrause** [comment]
@simaishi sure and no problem. Even (and hopefully we do) after we implement a label system, the reviewer/merger can always request a particular PR be added, if they forgot to add the changelog label at merge time.

I'll add the PRs you identified to this PR.

In the case of PR277, there was no milestone set. It never showed up on my list because of that omission. PR276 was excluded since it has a label of dependencies. (I'll add the list of labels that are excluded to the process doc)

**Pull request merged**